### DISCUSSION TITLE
Separation of interview question by different categories.

### DISCUSSION THREAD

**Discussion by Hasib-cirkut** [TamimEhsan/interview-questions-bangladesh#22] [Ideas]
Hi and Thanks for the initiative. 🚀 

Currently, the structure of the interview questions are only serialised by companies. 
But each of these companies have different positions. 
For example: Chaldaal has a Engineering dept. along with a Design dept and definitely a sales dept.(Ugh those random phone calls), 
Currently there is no way to separate these different Job posts. 
Even if we only consider the Engineering dept, you have SWE, QA, DevOps, Linux Admin and IT. 

So my idea for a feature is:
The contributors shall contribute to the Job post they applied to. 
For example: If any candidate applied to TSE, he should contribute to the ChalDaals TSE section. The same goes for the other positions. 

The folder structure could look like this:

\Chaaldaal
     -  trainee_swe.md
     - senior_swe.md
     - senior_sales_manager.md
     
An example of the side bar is given here:
![image](https://github.com/user-attachments/assets/c8ab0cef-ef27-478f-9623-d45eb2f6418d)

Let me know your thoughts and queries. If you are okay with, I can work and contribute to this feature.
Thanks and 🖖

**Comment by TamimEhsan** [comment]
Yes, That is a really nice idea. It will make the each categories cleaner. As a test run we can work with `Therap BD` as it has two seperate position there already. The other company sections can stay as it is. If the new modification works well with `Therap BD` we can roll out this feature to others too.  

Also, before making a contribution make sure you have been registered in hacktoberfest.

**Reply by Hasib-cirkut** [reply]
Hi. 
Thanks for the response. 
I agree with test run flow you suggested. I'll get started with a PR. 
Thanks. 

N.B: Yes I have registered in hacktoberfest with github. Do I have to do anything else extra?
